lib/sinatra/assetpack/helpers.rb

```diff
@@ -38,7 +38,7 @@ def show_one_asset_pack(type, name, options={})
         pack = settings.assets.packages["#{name}.#{type}"]
         return ""  unless pack
 
-        if settings.environment == :production
+        if settings.assets.compress_environments.member? settings.environment
```

Comment:
I like this patch. I don't think `****_environments` is too much of a concern as the only code branching is here. It's fairly common to want to enable/disable compression in different environments, this is a good solutions for that.